src/main/java/org/folio/dew/batch/acquisitions/edifact/CompositePOConverter.java

```diff
@@ -37,8 +37,9 @@ public void convertPOtoEdifact(EDIStreamWriter writer, CompositePurchaseOrder co
     messageSegmentCount++;
     writeOrderDate(compPO, writer);
 
+    String shipToAddress = configurationService.getAddressConfig(compPO.getShipTo());
```

Comment:
does `configurationService` have cache? not the best idea to make request for each order

Reply:
Possibly this class: `ConfigurationEntriesCache`

Reply:
Good point, Caching technique used, Sprint boot has cache feature, this feature has been used. Thanks